src/admin-views/notices/tribe-bf-general.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * The Black Friday admin notice.
+ *
+ * @since TBD
+ *
+ * @var string $icon_url The local URL for the notice's image.
+ * @var string $cta_url The short URL for black friday.
+ */
+?>
+<div class="tribe-marketing-notice">
+	<div class="tribe-marketing-notice__icon">
+		<img src="<?php echo esc_url( $icon_url ); ?>"/>
+	</div>
+	<div class="tribe-marketing-notice__content">
+		<h3>Save 40% on Every. Single. Plugin.</h3>
```

Comment:
Confirming -- not interested in translations, correct?

Reply:
Correct! It will likely change each year in some capacity, so forcing folks to translate a splash-in-the-pan marketing message feels less than ideal.